docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/pod-security-standards.md

```diff
@@ -2,67 +2,60 @@
-
 :::
 
 ## Pod Security Admission Configuration Templates
 
 Rancher comes with two Pod Security Admission Configuration (PSAC) templates that you can assign to a cluster:
-- `rancher-privileged`: This is the most permissive configuration and doesn't restrict the behavior of any pod.
-- `rancher-restricted`: This policy follows current pod-hardening best practices.
-
+- `rancher-privileged`: This is the most permissive configuration and does not restrict the behavior of any pod, allowing for known privilege escalations. This policy contains no exemptions.
+- `rancher-restricted`: This policy defines a heavily restricted configuration, based on current pod hardening best practices. This policy contains namespace level exemptions for Rancher components, as described below.
 
 If you are a Rancher administrator or have restricted administrator privileges, you can customize restrictions and permissions by creating additional PSAC templates, or by editing existing templates.
 
 :::caution
-
 If you edit an existing PSAC template while it is still in use, it *will* affect all clusters that have been assigned that template.
-
 :::
 
-If you want to allow users other than the Rancher administrator to manage PSAC templates, you can bind the user to a role that grants the user all verbs ("*") on `management.cattle.io/podsecurityadmissionconfigurationtemplates`.
+If you want to allow users other than the Rancher administrator to manage PSAC templates, you can bind the user to a role that grants the user all verbs (`"*"`) on `management.cattle.io/podsecurityadmissionconfigurationtemplates`.
 
 :::warning
-
 Any user that is bound to the above permission will be able to change the restriction levels on all managed clusters which use this template, including ones they have no permissions on.
-
 :::
 
 
-### Rancher On PSA-restricted Clusters
+### Rancher on PSA-restricted Clusters
 
-When you run Rancher on a Kubernetes cluster that enforces the `rancher-restricted` PSS policy by default, you'll need to exempt the following namespaces. This prevents the policy from blocking Rancher system pods:
+When you run Rancher on a Kubernetes cluster that enforces a restrictive PSS policy by default, such as the `rancher-restricted` PSS policy, you will need to exempt the following namespaces, otherwise the policy might prevent Rancher system pods from properly running.
 
-Here is the list of the namespaces that require an exemption:
+
+- `ingress-nginx`
 - `kube-system`
-- `kube-public`
 - `cattle-system`
-- `cattle-alerting`
-- `cattle-logging`
-- `cattle-prometheus`
-- `ingress-nginx`
-- `cattle-global-data`
-- `cattle-istio`
-- `kube-node-lease`
-- `cert-manager`
-- `cattle-global-nt`
-- `security-scan`
+- `cattle-epinio-system`
 - `cattle-fleet-system`
-- `cattle-fleet-local-system`
-- `calico-system`
-- `tigera-operator`
-- `cattle-impersonation-system`
-- `rancher-operator-system`
```

Comment:
Can you clarify why all of these were removed ? Would like @crobby to confirm changes as well before we merge.

Reply:
Removed because they are not in the list of ns currently exempt in the policy that we provide in Rancher - https://github.com/rancher/rancher/blob/release/v2.7/pkg/data/management/podadmissionconfigurationtemplate_data.go#L12-L33.

CC @crobby

Reply:
Ahh, that is because there are two lists. The template is for all downstream clusters and includes things like feature charts namespaces. There is a second list around system namespaces which is meant for local cluster: https://github.com/rancher/rancher/blob/release/v2.7/pkg/settings/setting.go#L27
Something like "cattle-impersonation-system" would only ever be on local.

Reply:
Though @crobby can you remove `rancher-operator-system`, the operator has been removed completely in 2.7.

Reply:
Yes, that all makes sense to me.

Reply:
@crobby and @cbron if you are fine with the updates, can you please approve the PR?

Reply:
@cbron and @crobby I updated the namespace exemption list, based on our offline discussion, through a separate commit to make it easier for review. PTAL.